src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable{
```

Comment:
The class 'Figure' is correctly set as abstract since it represents a generic concept from which specific figures will inherit. Make sure to use abstract classes and interfaces appropriately.